CFEScraping/index.js

```diff
@@ -0,0 +1,13 @@
+const puppeteer = require('puppeteer');
+
+(async () => {
+    const browser = await puppeteer.launch();
+    const page = await browser.newPage();
+
+    await page.goto('https://msc.cfe.mx/Aplicaciones/NCFE/Concursos/');
```

Comment:
Need to use a const to replace plain text, is impractical have the URL hardcoded. can be `const SCRAP_URL ,  const URL , etc.`